x-pack/plugin/sql/sql-action/src/main/java/org/elasticsearch/xpack/sql/action/SqlQueryRequest.java

```diff
@@ -33,12 +33,14 @@
     private static final ObjectParser<SqlQueryRequest, Void> PARSER = objectParser(SqlQueryRequest::new);
     static final ParseField COLUMNAR = new ParseField("columnar");
     static final ParseField FIELD_MULTI_VALUE_LENIENCY = new ParseField("field_multi_value_leniency");
+    static final ParseField INDEX_INCLUDE_FROZEN = new ParseField("index_include_frozen");
```

Comment:
@bpintea this is a new parameter that drivers can pass in to allow all queries inside one connection to include frozen indices. If not, the user would have to manually mention this through `INCLUDE FROZEN` and `FROZEN`.

Reply:
Opened https://github.com/elastic/elasticsearch-sql-odbc/pull/151. Note that the parameter is `false` by default.